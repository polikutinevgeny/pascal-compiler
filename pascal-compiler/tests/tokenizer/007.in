2343 2343 f 2343f