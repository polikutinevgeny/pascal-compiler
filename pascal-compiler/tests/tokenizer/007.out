Line |Pos  |Type        |Subtype                  |Value                              |Source                                            
----------------------------------------------------------------------------------------------------------------------------------------------
1    |1    |Constant    |IntegerConstant          |2343                               |1                                                 
1    |6    |Constant    |IntegerConstant          |2343                               |1                                                 
1    |11   |Identifier  |Identifier               |f                                  |1                                                 
1    |13   |Constant    |IntegerConstant          |2343                               |1                                                 
1    |17   |Identifier  |Identifier               |f                                  |1                                                 
